### DISCUSSION TITLE
Multi gpus resume error

### DISCUSSION THREAD

**Discussion by minhoooo1** [Lightning-AI/pytorch-lightning#12458] [DDP / multi-GPU / multi-node]
When I use multi gpus to resume ，the following error reports are generated : Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cuda:1! May be I should change map_location ,but where can i change in pytorch-lighning.

**Comment by mBerezovskyy** [comment]
@minhoooo1 did you find the solution?

**Comment by minhoooo1** [comment]
Nope.&nbsp;I haven't paid attention to this issue for a long time






------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年2月7日(星期三) 上午7:07
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [Lightning-AI/pytorch-lightning] Multi gpus resume error (Discussion #12458)





 
@minhoooo1 did you find the solution?
